Travel-list/src/components/FormSection.jsx

```diff
@@ -1,22 +1,49 @@
+// FormSection.jsx
+import { useState } from "react";
 import classes from "./FormSection.module.css";
-const FormSection = () => {
+
+const FormSection = ({ handleAddItem }) => {
+  const [quantity, setQuantity] = useState(1);
+  const [itemInput, setItemInput] = useState("");
+
+  const addItem = () => {
+    const newItem = {
+      Quantity: quantity,
+      Item: itemInput,
+    };
+    handleAddItem(newItem);
+    reset();
+  };
+
+  const reset = () => {
```

Comment:
You are defining reset at line 18 but calling it at line 15.. It shouldn't work, right? Cause, arrow functions are not hoisted.. Is it working?